hw/dv/sv/spi_agent/spi_agent.core

```diff
@@ -9,6 +9,7 @@ filesets:
     depend:
       - lowrisc:dv:dv_utils
       - lowrisc:dv:dv_lib
+      - lowrisc:dv:cryptoc_dpi:0.1
```

Comment:
Is this DPI really needed for the SPI agent? I think it gets used in `hw/top_darjeeling/dv/env/seq_lib/chip_sw_dma_spi_hw_handshake_vseq.sv` but not in one of the `spi_device` vseqs, right? So shouldn't this DPI be added to `hw/top_darjeeling/dv/env/chip_env.core`?

Reply:
This got marked as resolved but the still seems to be in the wrong place?

Reply:
I removed that dependency completely as it already gets pulled in.